Assets/MixedRealityToolkit/Definitions/Devices/MixedRealityControllerVisualizationProfile.cs

```diff
@@ -215,7 +215,7 @@ private bool SettingContainsParameters(MixedRealityControllerVisualizationSettin
         {
             return setting.ControllerType != null &&
                 setting.ControllerType.Type == controllerType &&
-                (setting.Handedness == hand || setting.Handedness == Handedness.Both);
+                setting.Handedness.HasFlag(hand);
```

Comment:
Will this work with `Handedness.None` hands, since `None = 0 << 0`?

Reply:
I believe right now, HasFlag will always be true for None. 

I think larger question is if this function/profile should even accept None as a valid input? @keveleigh thoughts?

Reply:
All of the visualization profile entries are either `Left` or `Right` at the moment, I'm pretty sure.
But if `None` gets passed in, we'll get something returned with this change (whereas previously it'd return false). Thinking about it more, I'm not sure if there's a current case where this would happen though.